hole/rock-paper-scissors-spock-lizard.go

```diff
@@ -27,5 +27,8 @@ func rockPaperScissorsSpockLizard() []Scorecard {
 		{"🦎✂", "✂ decapitates 🦎"},
 		{"🦎🖖", "🦎 poisons 🖖"},
 		{"🦎🦎", "Tie"},
-	}))
+	})
+
+	mid := len(tests) / 2
+	return outputTests(tests, tests[:mid], tests[mid:])
```

Comment:
There's a [helper func](https://github.com/code-golf/code-golf/blob/fa4290c9c7970370055cede1390f6b779b1d170b/hole/hole.go#L34) for this.